src/test/scala/com/goyeau/kubernetes/client/api/ConfigMapsApiTest.scala

```diff
@@ -41,4 +39,10 @@ class ConfigMapsApiTest
   override def modifyResource(resource: ConfigMap) =
     resource.copy(metadata = Option(ObjectMeta(name = resource.metadata.flatMap(_.name))), data = data)
   override def checkUpdated(updatedResource: ConfigMap) = updatedResource.data shouldBe data
+
+  override def deleteApi(namespaceName: String)(implicit client: KubernetesClient[IO]): Deletable[IO] =
```

Comment:
added a pair of methods to get specific return type to access their API. I could not find better way to expose the Deletable and Watchable methods.